src/mozillavpn.cpp

```diff
@@ -1880,14 +1875,6 @@ void MozillaVPN::registerInspectorCommands() {
         AddonManager::instance()->fetch();
         return QJsonObject();
       });
-
-  InspectorHandler::registerCommand(
-      "force_backend_failure", "Force a backend failure", 0,
```

Comment:
There is actually a force_heartbeat_command aswell, so both did the same things 🫠